src/rtt-gazebo-robot-sim.cpp

```diff
@@ -3,6 +3,8 @@
 #include <string>
 #include <fstream>
 #include <streambuf>
+#include <gazebo-7/gazebo/sensors/sensors.hh>
```

Comment:
Same as above:
Change ```#include <gazebo-7/gazebo/sensors/ForceTorqueSensor.hh>```
into this ```#include <gazebo/sensors/ForceTorqueSensor.hh>```

Because if we hardcode gazebo-7, then we will have problems once we change to another gazebo version in the future.